cmake/modules/packages/FindOGDI.cmake

```diff
@@ -12,28 +12,50 @@
 #    OGDI_VERSION
 #
 
-find_path(OGDI_INCLUDE_DIR ecs.h PATH_SUFFIXES ogdi)
+find_package(PkgConfig QUIET)
+if(PKG_CONFIG_FOUND)
+  pkg_check_modules(PC_OGDI QUIET OGDI)
+  set(OGDI_VERSION_STRING ${PC_OGDI_VERSION})
+  set(OGDI_INCLUDE_DIRS ${PC_OGDI_INCLUDE_DIRS})
+endif()
+
+find_path(OGDI_INCLUDE_DIR
+          NAMES ecs.h rpc/rpc.h
+          HINTS ${PC_OGDI_INCLUDE_DIRS})
```

Comment:
shouldn't the PATH_SUFFIXES ogdi should be kept (for non-pkgconfig cases)

Reply:
Sounds reasonable. I'll test it when I test the non-pkgconfig, cmake build.